src/openrct2/rct12/RCT12.cpp

```diff
@@ -752,6 +752,13 @@ void RCT12PathElement::SetIsBlockedByVehicle(bool isBlocked)
     }
 }
 
+bool RCT2TrackTypeIsBooster(uint8_t rideType, uint16_t trackType)
```

Comment:
Should be in `RCT2.cpp`/h, as I said [here](https://github.com/OpenRCT2/OpenRCT2/pull/13857#discussion_r565632018)